configs/smhtt_ul/config_ff_corr_tt.yaml

```diff
@@ -0,0 +1,17 @@
+# This is an example config containing information for the FF corrections
+
+workdir_name: "full_set_v1"
+era: "2018"
+channel: "tt" # options are et, mt, tt
+
+generate_json: True
+
+target_process:
+    QCD:
+        non_closure:
+            lep_pt:
+                var_dependence: "pt_1"
+                var_bins: [30,35,40,45,50,55,60,70,80,100]
```

Comment:
I'm not sure about the SM analysis, but in NMSSM the pT of the tailing had. tau and m_vis are corrected for the tt channel. I'm not totally sure if these variables will work out of the box, but I will add them if not.

Reply:
Basically resolved based on other comment.